pkg/config/default.go

```diff
@@ -226,6 +226,7 @@ func defaultConfig() (*Config, error) {
 			BaseHostsFile:       "",
 			CgroupNS:            cgroupNS,
 			Cgroups:             getDefaultCgroupsMode(),
+			ContainerNameAsHostName: false,
```

Comment:
this will need to be formatted correctly, i.e. gofmt should do it here.
However it should also be safe to drop this line as the default is already false. I don't think we have a strong rule that every default must be listed here although it seems most are

Reply:
I was just going by the example you pointed me to :)

What's the policy on changing formatting on unrelated lines?  When I initially saved the file, vscode realiged the entire block so the values lined up but I backed that out because it's harder to see what was a functional change vs a whitespace change.

Reply:
> I was just going by the example you pointed me to :)

I know, though in that case the default was true so it must be there. Anyway setting this here is fine you just need to fix the formatting.

> What's the policy on changing formatting on unrelated lines? When I initially saved the file, vscode realiged the entire block so the values lined up but I backed that out because it's harder to see what was a functional change vs a whitespace change.

Just do it the in the same commit. It is a bit more annoying for git blame and such but the consistent formatting makes it more readable in the long term.